### DISCUSSION TITLE
Editing the suricata.yaml file

### DISCUSSION THREAD

**Discussion by azxqw** [Security-Onion-Solutions/securityonion#2712] [Unsupported Versions]
Hi,
I want to edit my suricata so itll know the various servers in my network - i.e DC servers and such
I know that the suricata.yaml has the settings I need to edit, however every time i edit and restart suricata the setting are back to defualt, and I dont know how to add said settings to the salt config files.
help would be greatly appreciated.

**Comment by TOoSmOotH** [comment]
https://docs.securityonion.net/en/2.3/suricata.html#configuration

**Reply by techstartupexplorer** [reply]
that documentation doesn't say much no instruction how to edit the config using Salt

**Comment by InfosecGoon** [comment]
We actually have a video on how to set up these Suricata variables and tune rules with them:

https://www.youtube.com/watch?v=8TfnocUuQnM
